src/m365/spo/commands/knowledgehub/knowledgehub-get.ts

```diff
@@ -0,0 +1,85 @@
+import commands from '../../commands';
+import config from '../../../../config';
+import request from '../../../../request';
+import SpoCommand from '../../../base/SpoCommand';
+import {
+  CommandError
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import { ContextInfo, ClientSvcResponse, ClientSvcResponseContents } from '../../spo';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions { }
```

Comment:
Since we're not introducing any specific options, we can get rid of this interface